css/components/hero.css

```diff
@@ -0,0 +1,65 @@
+.container.no-padding-bottom {
+  padding-bottom: 0;
+}
+
+.row.visible-flex.hero-block-position {
+  align-content: space-around !important;
+  justify-content: center;
+  flex-wrap: wrap;
+  height: 720px;
+}
+
+.container.hero {
+  display: flex;
+  align-items: center;
+  height: 750px;
+}
+
+.row.small-margin-bottom {
+  margin-bottom: 0;
+}
+
+.align-self {
+  align-self: center !important;
+}
+
+/* sm */
+@media (min-width: 575px) {
+  .visible-flex.hero-block-position {
+    align-content: space-around;
+    justify-content: center;
+    flex-wrap: wrap;
+  }
```

Comment:
toliau matau kartojasi tas pats kodas is esmes, jeigu nori isscentruoti turini, kad jie dviese susiglude sekcijos centre butu, uztenka parasyti align-items:center (ne content) ir justify-content:center. nelabai suprantu ka nori padaryti su tuo space around sitoj vietoj ir bet kuriuo atveju atsidarius inspekta ir juos visus tris (29, 30, 31 eilute) uzkomentavus nieks nesikeicia, tai jie neveikia